service/src/main/java/bio/terra/pipelines/dependencies/sam/SamClient.java

```diff
@@ -30,7 +32,7 @@ private ApiClient getApiClient() {
     var okHttpClientWithTracing =
         this.okHttpClient
             .newBuilder()
-            .addInterceptor(new OkHttpClientTracingInterceptor(Tracing.getTracer()))
+            .addInterceptor(new OkHttpClientTracingInterceptor(openTelemetry))
```

Comment:
can we do this for wds as well? i believe they also use `okHttp` to make their client

Reply:
definitely we can - are you ok if that goes in the tracing ticket (TSPS-107) rather than this PR?